assets/scripts/whichx.js

```diff
@@ -158,18 +201,29 @@ function WhichX(config) {
         return typeChance * (type.tcount / total.tcount);
     }
 
-    // A non-zero prior estimate to prevent 0 based probability calculations.
+    /**
+     * A non-zero prior estimate to prevent 0 based probability calculations.
+     * @returns {number} The non-zero probablity
+     */
     function mEstimate() {
         var total = typesMap.total;
         return 1 / (total.wordTotal * 100);
     }
 
-    // Process the description into an array of pertinent standardized lower case words.
```

Comment:
This function is not available to call on the `WhichX` object, so the JSDoc here may not be all the helpful. Can we remove the docs to keep file size down?

Reply:
See above comment